### ISSUE TITLE
Horizontal line at top in fullscreen with horizontal tab feature

### ISSUE THREAD

**Issue by simonhong** [brave/brave-browser#33927]
## Description 

I can see white horizontal line at top like below. It's more easily visible with light theme.
<img width="1500" alt="image" src="https://github.com/brave/brave-browser/assets/6786187/d1bce0f9-4733-4b2e-9901-709a8265f96a">

## Steps to Reproduce

   1. Launch browser
   2. Enters into full screen mode from any youtube clip
   3. Check horizontal line like above screen shot

## Actual result:

horizontal line

## Expected result:
No horizontal line

## Reproduces how often: 

Easy

## Version/Channel Information:

- Can you reproduce this issue with the current release? 
- Can you reproduce this issue with the beta channel? Yes
- Can you reproduce this issue with the nightly channel? Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? 
- Does the issue resolve itself when disabling Brave Rewards? 
- Is the issue reproducible on the latest version of Chrome? 

## Miscellaneous Information:

Can see this on macOS. Windows doesn't have this issue.
Not tested with linux yet.

cc @rebron @kjozwiak @zenparsing

**Comment by stephendonner** [comment]
## Verified `IN-PROGRESS` using

**Comment by kjozwiak** [comment]
The above requires `1.61.75` or higher for `1.60.x` verification. Removing the `QA Pass-macOS` label as we'll need to re-check the above via `1.61.x`. However, used https://github.com/brave/brave-browser/issues/33927#issuecomment-1800423613 as the Nightly verification to get the above uplifted into `1.61.x` as per https://github.com/brave/brave-core/pull/20946#pullrequestreview-1728387356.

**Comment by LaurenWags** [comment]
Verification in progress with

```
Brave | 1.61.75 Chromium: 119.0.6045.134 (Official Build) beta (x86_64)
-- | --
Revision | 1b1191cb2465b2b90a5e8fea4a63aed1cfca47f5
OS | macOS Version 13.6.1 (Build 22G313)
```

**Comment by LaurenWags** [comment]
Verified with

```
Brave	1.61.75 Chromium: 119.0.6045.134 (Official Build) beta (arm64) 
Revision	1b1191cb2465b2b90a5e8fea4a63aed1cfca47f5
OS	macOS Version 14.2 (Build 23C5041e)
```

Reproduced the issue using STR from https://github.com/brave/brave-browser/issues/33927#issue-1962537151 and `1.61.74` with light theme as suggested.
Verified fix using same STR and `1.61.75` with light theme.

1.61.74 | 1.61.75
--- | ---
![74](https://github.com/brave/brave-browser/assets/28145373/9509b322-8f8d-486c-8309-8f76a73e83f4) | ![75](https://github.com/brave/brave-browser/assets/28145373/3f2fc768-fe06-41b6-ab8d-f05876d5241f)
